lib/psp/manager.rb

```diff
@@ -112,13 +127,35 @@ def calculate_plugin_runners_allocation
       end
     end
 
+    def calculate_gem_runners_allocation
+      return empty_gem_runners_allocation if @gm_runners_count.zero?
+
+      allocation = @gems.flat_map(&:directory)
+                     .in_groups(@gm_runners_count, false)
+                     .reject(&:empty?)
```

Comment:
Use 2 (not 15) spaces for indenting an expression in an assignment spanning multiple lines.